metrics-core/src/main/java/io/dropwizard/metrics/Histogram.java

```diff
@@ -53,4 +58,15 @@ public long getCount() {
     public Snapshot getSnapshot() {
         return reservoir.getSnapshot();
     }
+
+    @Override
+    public String toString() {
+        final ByteArrayOutputStream out = new ByteArrayOutputStream();
+        this.getSnapshot().dump(out);
+        try {
+            return out.toString(Charset.defaultCharset().name());
```

Comment:
should we just use `StandardCharsets.UTF_8` here to avoid having to deal with `UnsupportedEncodingException`?